Source/StreetMapImporting/OSMFile.cpp

```diff
@@ -349,6 +349,20 @@ bool FOSMFile::ProcessAttribute( const TCHAR* AttributeName, const TCHAR* Attrib
 					// @todo: Add support for interpreting unit strings and converting the values
 				}
 			}
+			else if (!FCString::Stricmp(CurrentWayTagKey, TEXT("building:levels")))
+			{
+				// Check for definition of building level count
+				// This value should contain no spaces? But just leave it in to be sure.
```

Comment:
We can go ahead and remove this check for units.  The spec says there won't be any.